simple-no-auth/src/test/java/com/fga/example/OpenFgaContainerConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.fga.example;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+
+@TestConfiguration(proxyBeanMethods = false)
+class OpenFgaContainerConfiguration {
+
+	@Bean
+	GenericContainer<?> openFgaContainer(DynamicPropertyRegistry registry) {
+		var result = new GenericContainer<>("openfga/openfga:latest")
```

Comment:
I would highly recommend pinning the version to avoid breakages (`:latest` is like `-SNAPSHOT` in Maven):
```suggestion
		var result = new GenericContainer<>("openfga/openfga:v1.4.3")
```